blog/app/helpers/symbolize_helper.rb

```diff
@@ -0,0 +1,25 @@
+module SymbolizeHelper
+    extend self
+  
+    def symbolize_recursive(hash)
```

Comment:
1. Where is this used?
2. If you're using Rails, you get it covered by activesupport - https://apidock.com/rails/v3.2.13/Hash/symbolize_keys